### DISCUSSION TITLE
"Made with Publii" Icon

### DISCUSSION THREAD

**Discussion by johnwoltman** [GetPublii/Publii#1568] [General]
I know this is going to date me but I wanted an old-fashioned 88x31 button for my Publii site, so I made one:

![made_with_publii](https://github.com/GetPublii/Publii/assets/2498295/841cc5e7-c16f-4d4a-8162-94d17c3264b3)

If you're old (or old at heart) and feel like using it, please do so.

**Comment by candidexmedia** [comment]
Nice, thank you for sharing!
